README.md

```diff
@@ -486,6 +485,7 @@ Table of Contents
   * [Octopus Deploy](https://octopus.com) - Automated deployment and release-management. Free for <= 10 deployment targets.
   * [scalr.com](https://scalr.com/) - Remote state & operations backend for Terraform with full CLI support, integration with OPA and a hierarchical configuration model. Free up to 5 users.
   * [semaphoreci.com](https://semaphoreci.com/) — Free for Open Source, 100 private builds per month
+  * [Serverless Cloud](https://www.serverless.com/cloud) - Serverless Cloud lets you build Serverless APIs, DBs and Storage by using infrastrucure _from_ code approach(no yaml, no infrastructure configuration). The product is currently under public preview. 
```

Comment:
I dont think this is a CI/CD service, they have a small aspect of that but predominantly its a PaaS

Reply:
Remaining point here

Reply:
Sorry,I didn't find this feature when I first experienced it. Now I found it

Reply:
I recover it!